tests/macos/xcodeproj/NonArcObject.h

```diff
@@ -1,2 +1,2 @@
-@interface NonArcObject
+@interface NonArcObject : NSObject
```

Comment:
Is this intentional? This will make it an ARC object now that you are specifying that inherits from `NSObject`?

Reply:
I believe whether it is ARC or not is defined by compiler flag `-fno-obj-arc`, @amberdixon can correct me here? 
Also it's weird that my previous PR makes CI fail on this...

Reply:
What we are really trying to test here is that the builder process treats this file like a non-arc object and adds the no-objc-arc flag. So with or without this change, our tests and builds should work as expected.

Even before ARC came along, NSObject was used I believe, so I'm not sure inheriting NSObject causes the file to become ARC ... i googled around a little, can you share any links/docs that indicate that that would be the case?

Reply:
For whatever reason, I thought ARC was managed at the object level... but brushing up on the [documentation for ARC](https://clang.llvm.org/docs/AutomaticReferenceCounting.html#general) (it's been a while) reminded me it just inserts retain/release for you during compilation. Sorry for the confusion.

Reply:
yeah i have not seen `retain` `release` for ages :)